docs/static_site/src/.htaccess

```diff
@@ -22,6 +22,14 @@ RewriteOptions AllowNoSlash
   
 </IfModule>
 
+# TODO temporary fix for issue #18604
+Redirect 302 /api/r/docs/api/R-package/build/mxnet-r-reference-manual.pdf https://s3.amazonaws.com/mxnet-prod/docs/R/mxnet-r-reference-manual.pdf
```

Comment:
There's a much more recent version of this and it's like 1000 pages long...

Reply:
Updated to the newest version.